flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSource.java

```diff
@@ -80,23 +83,27 @@ public HiveTableSource(JobConf jobConf, ObjectPath tablePath, CatalogTable catal
 	private HiveTableSource(JobConf jobConf, ObjectPath tablePath, CatalogTable catalogTable,
```

Comment:
Maybe it's better to add some method comments here to indicate its effect on projection or partition-prunning.

Reply:
Sure